src/buildHTML.js

```diff
@@ -1034,54 +1008,51 @@ groupTypes.katex = function(group, options) {
         ["mord", "katex-logo"], [k, a, t, e, x], options);
 };
 
+const makeLineSpan = function(className, options) {
+    const baseOptions = options.havingBaseStyle();
+    const line = makeSpan(
+        [className].concat(baseOptions.sizingClasses(options)),
```

Comment:
Are adding the sizing classes necessary here?  The span has no children and we're already setting its height explicitly.